iOS_SDK/OneSignalSDK/OneSignalCoreMocks/Extensions/NSDictionary+UnitTests.swift

```diff
@@ -1,13 +1,62 @@
 extension NSDictionary {
-    func contains(key: String, value: Any) -> Bool {
+    /*
+     This method goes one level deep into dictionaries
+     */
+    private func contains(key: String, value: Any) -> Bool {
         guard let dictVal = self[key] else {
             return false
         }
-
-        return equals(dictVal, value)
+        if let value = value as? [String: Any],
+        let dictVal = dictVal as? NSDictionary {
```

Comment:
Nit: This `if let` formating seems very odd, is this meet standard Swift linting?

Reply:
That's my bad, I did not align the `let`s.

Ideally this looks most correct to me:
```swift
if let value = value as? [String: Any],
   let dictVal = dictVal as? NSDictionary 
{
    return dictVal.contains(value)
} else {
    return equals(dictVal, value)
}
```

But running the swift auto-linter `swiftlint --fix`, the opening `{` does move to next line:
```swift
if let value = value as? [String: Any],
   let dictVal = dictVal as? NSDictionary {
    return dictVal.contains(value)
} else {
    return equals(dictVal, value)
}
```

Thoughts? There's a lot of these patterns and we can decide a better format we want to use.

Reply:
@jkasten2 That's my bad, I did not align the `let`s, and the swiftlint auto-linter did not fix it.

I think this looks most clear to me:
```swift
if let value = value as? [String: Any],
   let dictVal = dictVal as? NSDictionary 
{
    return dictVal.contains(value)
} else {
    return equals(dictVal, value)
}
```

But running the swift auto-linter `swiftlint --fix`, it moves the opening `{` up:
```swift
if let value = value as? [String: Any],
   let dictVal = dictVal as? NSDictionary {
    return dictVal.contains(value)
} else {
    return equals(dictVal, value)
}
```

Thoughts? There's a lot of these type of `if let` or `guard let` in the codebase, and we can decide a better format we want to use.

Reply:
`swiftlint --fix` looks fine, I don't have any strong feels with that standard.

Original code really throw me off as; it was a combination of the where the 2nd `let` lined up with the `if` and not knowing you could use a `,` that allows multiple variables in the `if`.

Reply:
Sounds good to me. Fixed up the original commit.